### PR TITLE
Rails 3 / Rspec 2 compatibility

### PR DESCRIPTION
**Pull request by awebneck**
Hey guys, altered the plugin for rails 3 / rspec 2 compatibility - it's tested and working on those platforms, though backwards compatibility is (as one may expect) broken. Not sure if you want to bump a new branch for this or what, but just letting you know it's here.
- Jeremy

### CODE REVIEW

File: lib/object_daddy.rb [hunk 1]
```diff
@@ -33,7 +33,10 @@ module ObjectDaddy
           : const_get(@concrete_subclass_name).spawn(args)
       end
       generate_values(args)
-      instance = new(args)
+      instance = new
+      args.each_pair do |attribute, value|
+        instance.send("#{attribute}=", value)  # support setting of mass-assignment protected attributes
```
Comment (rick) [inline lib/object_daddy.rb:40]: This seems like maybe a feature addition, as opposed to being just part of the rails3/rspec2 upgrade.  T or F?

**Comment by rick** [comment]
Sorry it took so long to finally get a look at this.  @cardioid might want to eyeball it too.  Looks good, though it seems like there are things in here that aren't related to rails3/rspec2 -- could be that the branch point is different from what is current master?

File: lib/object_daddy.rb (hunk 1 above)
Comment (awebneck) [inline lib/object_daddy.rb:40]: T. I believe this came in from the repo I originally branched - not from me. Feel free to reject it.

**Comment by awebneck** [comment]
quite so - feel free to reject that args.each_pair stuff - I didn't put it there and I believe it came from the branch I originally forked (the branch from which the gem was produced)

**Comment by awebneck** [comment]
After speaking to the authors regarding gemification and maintenance, I'm closing this request, and am now maintaining (with the authors' permission) a rails 3+, ruby 1.9+ -compatible gem version on my branch. The gem is named "object-daddy" because some gentleman apparently deployed an older gem named "object_daddy", then subsequently disappeared. Attempts to contact him have been for naught, so taking over the originally-named gem is out of the picture.

**Comment by rick** [comment]
My name is Rick Bradley, and I approve this message.

**Pull request closed without merging**